docs/api/_common/api-types.mdx

```diff
@@ -0,0 +1,31 @@
+There are two main API types available in Ory:
+
+- Admin APIs, which are often used by the application backend, offer privileged access that's not available to end users. To use
+  Admin APIs, you need an Ory Network API Key.
+- Public APIs, which are often used by the application frontend, don't require an Ory Network API Key because:
+  - they don't require authorization and can be accessed by any client, for example, an application's login page
```

Comment:
```suggestion
  - they don't require authorization and can be accessed by any client, for example, an application's login page.
```

After some thought I decided it's better to add a period here - the main list item and this sub-item together make a full sentence.